iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/InjectionObservabilityAdderImpl.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtensionAdder;
+import com.powsybl.iidm.network.Injection;
+import com.powsybl.iidm.network.extensions.InjectionObservability;
+import com.powsybl.iidm.network.extensions.InjectionObservabilityAdder;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class InjectionObservabilityAdderImpl<I extends Injection<I>>
+        extends AbstractExtensionAdder<I, InjectionObservability<I>>
+        implements InjectionObservabilityAdder<I> {
+
+    private boolean observable;
```

Comment:
To be confirmed (@annetill @sylvlecl ?) but I think observable is optional and should then be a `Boolean` object and `null` if not defined.

Reply:
I think it's ok to have it not nullable. At the end of a state estimation we should be able to say for every element if it was observable or not.
In any case, if for any reason we cannot say it, I think it will be logical to not have the extension at all. In that case, the other values will not make sense either.